packages/devextreme/testing/tests/DevExpress.ui.widgets/tabPanel.tests.js

```diff
@@ -532,6 +532,21 @@ QUnit.module('focus policy', {
             });
         });
     });
+
+    QUnit.test('changing showNavButtons should not set focus class on wrapper in runtime', function(assert) {
+        const $tabPanel = $('#tabPanel').dxTabPanel({
+            items: ['item 1'],
+            showNavButtons: false,
+        });
+        const tabPanel = $tabPanel.dxTabPanel('instance');
+
+        $tabPanel.focusin();
+        $tabPanel.focusout();
+
+        tabPanel.option({ showNavButtons: true });
+
+        assert.strictEqual($($tabPanel).find(`.${MULTIVIEW_WRAPPER_CLASS}`).hasClass(FOCUS_STATE_CLASS), false);
```

Comment:
Why we should wrap $tabpanel?

Reply:
[d1ca377dccaea2cb377080118f0af3485cf3c0ec]